pkg/controller/clusterpool/clusterdeploymentexpectations.go

```diff
@@ -13,21 +14,19 @@ import (
 )
 
 func (r *ReconcileClusterPool) watchClusterDeployments(c controller.Controller) error {
-	handler := &clusterDeploymentEventHandler{
-		EventHandler: &handler.EnqueueRequestsFromMapFunc{
-			ToRequests: handler.ToRequestsFunc(
-				func(a handler.MapObject) []reconcile.Request {
-					cpKey := clusterPoolKey(a.Object.(*hivev1.ClusterDeployment))
-					if cpKey == nil {
-						return nil
-					}
-					return []reconcile.Request{{NamespacedName: *cpKey}}
-				},
-			),
-		},
-		reconciler: r,
+	h := &clusterDeploymentEventHandler{
+		handler.EnqueueRequestsFromMapFunc(
+			func(a client.Object) []reconcile.Request {
+				cpKey := clusterPoolKey(a.(*hivev1.ClusterDeployment))
+				if cpKey == nil {
+					return nil
+				}
+				return []reconcile.Request{{NamespacedName: *cpKey}}
+			},
+		),
+		r,
```

Comment:
struct is being initialized without Field names https://github.com/uber-go/guide/blob/master/style.md#use-field-names-to-initialize-structs